indico/legacy/webinterface/tpls/RoomBookingNewBookingPeriodWidget.tpl

```diff
@@ -208,6 +217,47 @@ ${ form.repeat_interval(type='hidden') }
             }
         }
 
+        function checkBookingLimit() {
+            var oneDay = 24*60*60*1000;
+            var startDate = $('#sDatePlace').datepicker('getDate');
+            var endDate = $('#eDatePlace').datepicker('getDate');
+            var selectedDaysPeriod = (endDate - startDate)/oneDay;
+            var selectedRooms = $("#roomselector").roomselector("selection");
+            var selectedNotLimitedRooms = _.clone(selectedRooms);
+            var limitedRoomsInvalid = [];
+            for (var i = 0; selectedDaysPeriod >= 0 && i < selectedRooms.length; i++) {
+                var roomBookingLimit = rooms[selectedRooms[i]].booking_limit_days;
+                if (roomBookingLimit) {
+                    selectedNotLimitedRooms.splice(selectedNotLimitedRooms.indexOf(selectedRooms[i]), 1);
+                    if (selectedDaysPeriod > roomBookingLimit) {
+                        limitedRoomsInvalid.push(rooms[selectedRooms[i]]);
+                    }
+                }
+            }
+            handleLimitedRoomsInvalid(limitedRoomsInvalid);
+            if (selectedNotLimitedRooms.length) {
+                var bookingDaysLimit = ${ booking_limit };
+                var islimitExceeded = endDate > startDate && selectedDaysPeriod > bookingDaysLimit;
+                $('.booking-limit-warning').toggleClass('hidden', !islimitExceeded);
+                $('#searchForm :submit').attr('disabled', islimitExceeded);
+            } else {
+                $('.booking-limit-warning').toggleClass('hidden', true);
+                $('#searchForm :submit').attr('disabled', !!limitedRoomsInvalid.length);
+            }
+        }
+
+        function handleLimitedRoomsInvalid(rooms) {
+            if (rooms.length) {
+                var warningMsgRooms = [];
+                $.each(rooms, function(i, room) {
+                    warningMsgRooms.push("'{0}' for more than {1} days".format(room.name, room.booking_limit_days));
+                });
+                var warningMsg = $T.gettext("It is not possible to book room {0}".format(warningMsgRooms.join(", ")));
```

Comment:
Can you add  a screenshot to this PR how the warning looks like when there are a few rooms where the restriction is not met?